pkg/ssh/exec.go

```diff
@@ -32,6 +31,9 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+var lostCharacter = make(chan []byte, 1)
```

Comment:
could we have a single channel?
if there are several reconnections, there could be several lost chars

Reply:
I will try these user cases and try to have just one channel